amqp-vertx/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+target/
```

Comment:
Merge this .gitignore, with repo's .gitignore which is at the root mq-dev-patterns directory.